index.js

```diff
@@ -94,9 +95,13 @@ function conventionalGithubReleaser(auth, changelogOpts, context, gitRawCommitsO
 
           var prerelease = semver.parse(version).prerelease.length > 0;
 
+          // conventional changelog don't parse
+          // owner of ghe reposity yet
+          var owner = parse(context.repository)[1]
```

Comment:
It won't when using ghe like repository =\
as mentioned in [get-pkg-repo/issues/3](https://github.com/stevemao/get-pkg-repo/issues/3)